docs/DeveloperGuide.md

```diff
@@ -1,34 +1,509 @@
-# Developer Guide
+# Developer Guide for PlanNUS
 
-## Design & implementation
+<table><tr><td><div style="text-align:center">
+    <img src="images/PlanNUSLogo.png" />
+</div></td></tr></table>
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+Before reading this document, you are recommended to read through the [user guide](https://ay2021s1-cs2113t-f12-1.github.io/tp/UserGuide.html).
 
+## Table of contents 
 
-## Product scope
-### Target user profile
+* Table of contents
```

Comment:
The table of contents is well done! I like how I can navigate to different sections by clicking on the title.